be/src/olap/storage_engine.cpp

```diff
@@ -495,10 +491,12 @@ void StorageEngine::clear_transaction_task(const TTransactionId transaction_id,
         // each tablet
         for (auto& tablet_info : tablet_infos) {
             // should use tablet uid to ensure clean txn correctly
-            TabletSharedPtr tablet = _tablet_manager->get_tablet(tablet_info.first.tablet_id, 
+            TabletSharedPtr tablet = _tablet_manager->get_tablet(tablet_info.first.tablet_id,
                 tablet_info.first.schema_hash, tablet_info.first.tablet_uid);
+            // The tablet may be dropped or altered, leave a INFO log and go on process other tablet
             if (tablet == nullptr) {
-                return;
+                LOG(INFO) << "tablet is no longer exist. tablet_id=" << tablet_info.first.tablet_id;
```

Comment:
LOG schema hash and tablet_uid to be more clear

Reply:
Done.